w3-account.md

```diff
@@ -226,17 +244,17 @@ However user may also add new delegations on one device and expect to have acces
 
 # Limitations
 
-Using delegation from specific authority as an authorization proof limits it to the contexts in which signing authority is trusted. It is reasonable compromise when receiver of the proof and issuer is the same authority (as is the case for web3.storage).
+Using delegation from specific authority as an authorization proof limits it to the contexts in which signing authority is trusted. It is a reasonable compromise when receiver and the issuer of the proof are the same authority (as is the case for a service like web3.storage).
 
-However we wish above described account system to be usable in global context, which we plan to accomplish by upgrading [authorization][] to use [UCAN mailto][] specification, so that email ownership could be verifiable without [email verification][] step.
+We will remove this limitation in the future the account system by upgrading [authorization][] to use the [UCAN mailto][] specification, so that email ownership may be verified without the [email verification][] step described above, allowing this accounts system to be used from in a global, permissionless context.
 
 # Related Notes
 
 ## Free provider
 
-web3.storage offers one "free provider" per account. It will be denied if `consumer` space is not specified or is the one that already has it.
+web3.storage offers one "free provider" per account. It will be denied if a `consumer` space is not specified or has already been applied.
```

Comment:
```suggestion
web3.storage offers one "free provider" per account. It will be denied if a `consumer` space is not specified or if account has already claimed it with a different `consumer` space.
```